lang/en.yml

```diff
@@ -1,19 +1,19 @@
 en:
-  Widget:
+  SilverStripe\Widgets\Extensions\WidgetPageExtension:
+    INHERITSIDEBAR: 'Inherit Sidebar From Parent'
+  SilverStripe\Widgets\Model\Widget:
     PLURALNAME: Widgets
     SINGULARNAME: Widget
-  WidgetArea:
+  SilverStripe\Widgets\Model\WidgetArea:
     PLURALNAME: 'Widget Areas'
     SINGULARNAME: 'Widget Area'
-  WidgetAreaEditor_ss:
+  SilverStripe\Widgets\Templates\SilverStripe\Widgets\Forms\WidgetAreaEditor_ss:
```

Comment:
These template pseudo namespaces need to be reverted to what they were so they match the `<%t` calls in the templates they reference :)